docs/source/netutils/configs/index.rst

```diff
@@ -9,4 +9,11 @@ Configs
     :members:
 
 .. automodule:: netutils.config.parser
-    :members:
\ No newline at end of file
+    :members:
+
+Edge Cases
+==============
+
+Fortinet Fortios Parser
+-----------------------
+- In order to support html blocks that exist in Fortios configurations, some preprocessing is executed, this is a regex that specifically grabs everything between quotes after the 'set buffer' sub-command. Its explicitly looking for double quote followed by a newline ("\n) to end the captured data.  This support for html data will not support any other html that doesn't follow this convention.
```

Comment:
```suggestion
- In order to support html blocks that exist in Fortios configurations, some preprocessing is executed, this is a regex that specifically grabs everything between quotes after the 'set buffer' sub-command. It's explicitly looking for double quote followed by a newline ("\n") to end the captured data.  This support for html data will not support any other html that doesn't follow this convention.
```

Reply:
Will fix the `It's` but the second is actually intended.